### DISCUSSION TITLE
Why isn't Turbowarp getting new extensions anymore?

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#1101] [Q&A] [answered]
When I first found Turbowarp extensions, there used to be new extensions every week. Now, there's something new every couple of months. Did something happen that I'm not aware of? I need answers.

**Comment by softedco** [comment]
There was a bit of drama involved but I guess that didn't affect the frequency of uploads
Also it's not easy to make something new once all simple things are already done, there is basically a plato on how much you can extend turbowarp with javascript and we are simply approaching it. There are API wrapper extensions though which have an entirely separate plato, but most of these are commercial or take more time to implement and maintain.

**Reply by ghost** [reply]
Like what drama?

**Reply by GarboMuffin** [reply]
The "drama" is unrelated and is not worth further discussion

**Comment by DNin01** [comment, marked as answer]
It's probably just that we've made extensions for so many areas already and that @GarboMuffin is extra busy at the moment.
